README.md

```diff
@@ -168,11 +170,11 @@ spec:
   embedding: default
   mode: remote
 
-# Create the database
-./cli/maestro-k vectordb create my_database.yaml
+# Create the database (using full path to CLI binary)
+/path/to/maestro-cli/commands vectordb create my_database.yaml
```

Comment:
I'm not sure this is the best way to write this, I would just put `maestro` and assume it's on the path according to it's own install steps

Ditto on the below instances